pkg/systemd/generate/containers.go

```diff
@@ -394,6 +394,10 @@ func executeContainerTemplate(info *containerInfo, options entities.GenerateSyst
 			info.IdentifySpecifier = true
 			runIx := -1
 			nameIx := -1
+			// Add systemd identify specifier next to the name value
+			// to set a name to the container according to the parameters passed to systemd.
+			// In case no --name set for that container, use ServiceName specified
+			// in the containerInfo struct.
```

Comment:
@vrothberg is this comment good enough? Maybe changing the code a little bit can help?

Reply:
Yes that helps but please create a dedicated function for that and add unit tests.

Reply:
OK no problem.

Reply:
DONE